ui/src/app/job-list/table/table.component.ts

```diff
@@ -249,4 +261,58 @@ export class JobsTableComponent implements OnInit {
       }
     }
   }
+
+  openBulkChangeDialog(): void {
+    this.resetBulkLabelFieldDefaults();
+    // figure out default values for bulk fields; if all jobs have the same label value,
+    // set that to the defualt; otherwise, set default to boolean false
+    for (let j = 0; j < this.selection.selected.length; j++) {
+      for (let f = 0; f < this.bulkLabelFields.length; f++) {
```

Comment:
Unless you explicitly need the index, I would iterate with "let ... of", i.e. `let f of this.bulkLabelFields`

Reply:
done